doc/telescope.txt

```diff
@@ -131,66 +138,56 @@ Available layout strategies include:
 
 
 layout_strategies.center()                        *layout_strategies.center()*
-
     Centered layout wih smaller default sizes (I think)
 
-       +--------------+
-       |    Preview   |
-       +--------------+
-       |    Prompt    |
-       +--------------+
-       |    Result    |
-       |    Result    |
-       |    Result    |
-       +--------------+
-
-
+        ++
+        |    Preview   |
+        ++
```

Comment:
is docgen stripping the `-` ?

Reply:
Something weird is going on yeah. Will look into that later. Because that is what the logging also shows :|
https://github.com/Conni2461/telescope.nvim/runs/1980018498?check_suite_focus=true#step:7:48

Reply:
tj just pushed a commit that fixes that. :D Retriggered CI. Works great now :D

Reply:
great!